src/frontend/src/test-e2e/alternativeOrigin.test.ts

```diff
@@ -0,0 +1,236 @@
+import { AuthenticateView, DemoAppView, ErrorView, WelcomeView } from "./views";
+import { FLOWS } from "./flows";
+import {
+  addVirtualAuthenticator,
+  addWebAuthnCredential,
+  getWebAuthnCredentials,
+  originToRelyingPartyId,
+  runInBrowser,
+  switchToPopup,
+  waitToClose,
+} from "./util";
+
+import {
+  II_URL,
+  TEST_APP_NICE_URL,
+  REPLICA_URL,
+  DEVICE_NAME1,
+  TEST_APP_CANISTER_ID,
+  TEST_APP_CANONICAL_URL,
+} from "./constants";
+
+test("Should not issue delegation when derivationOrigin is missing from /.well-known/ii-alternative-origins", async () => {
+  await runInBrowser(async (browser: WebdriverIO.Browser) => {
+    const authenticatorId1 = await addVirtualAuthenticator(browser);
+    await browser.url(II_URL);
+    const userNumber = await FLOWS.registerNewIdentityWelcomeView(
+      DEVICE_NAME1,
+      browser
+    );
+    await FLOWS.addRecoveryMechanismSeedPhrase(browser);
+    const credentials = await getWebAuthnCredentials(browser, authenticatorId1);
+    expect(credentials).toHaveLength(1);
```

Comment:
would it be worth to also check the content?

Reply:
ah this was just moved